WooCommerce/Classes/POS/Models/PointOfSaleAggregateModel.swift

```diff
@@ -149,6 +153,14 @@ extension PointOfSaleAggregateModel {
         paymentState = .card(.idle)
         cardPresentPaymentInlineMessage = nil
     }
+
+    private func trackCustomerInteractionStarted() {
```

Comment:
;nit

We can put in these trackings into a different extension for code organization

Reply:
Updated! https://github.com/woocommerce/woocommerce-ios/pull/15138/commits/66141e4036220d56fbcaab670e11f6e669abe7c8